### ISSUE TITLE
Connections and some nodes are gone after update to commit 15b26ec4829f64418e47cd99410415604fe16ac7

### ISSUE THREAD

**Issue by ijisthee** [acdamiani/schema#14]
Hi @acdamiani 

I've updated Schema AI via import from git (before I downloaded it from Asset Store).

When I open my Graph, I get this error message / stack trace:

```
Broken text PPtr in file(Assets/Prefabs/Enemies/BaseAIGraph.asset). Local file identifier (86440366174919129) doesn't exist!
UnityEngine.GUIUtility:ProcessEvent (int,intptr,bool&)
```

```
Exception thrown while invoking [OnOpenAssetAttribute] method 'SchemaEditor.NodeEditor:Open (int,int)' : NullReferenceException: Object reference not set to an instance of an object
SchemaEditor.Internal.ComponentSystem.Components.ConditionalComponent.Create (SchemaEditor.Internal.ComponentSystem.CreateArgs args) (at ./Library/PackageCache/com.acdamiani.schema@15b26ec482/Editor/Canvas/Components/ConditionalComponent.cs:177)
SchemaEditor.Internal.ComponentCanvas.Create[T] (SchemaEditor.Internal.ComponentSystem.CreateArgs args) (at ./Library/PackageCache/com.acdamiani.schema@15b26ec482/Editor/Canvas/ComponentCanvas.cs:99)
SchemaEditor.NodeEditor.RebuildComponentTree () (at ./Library/PackageCache/com.acdamiani.schema@15b26ec482/Editor/Window/NodeEditorGUI.cs:390)
SchemaEditor.NodeEditor.Open (Schema.Graph graphObj) (at ./Library/PackageCache/com.acdamiani.schema@15b26ec482/Editor/Window/NodeEditor.cs:142)
SchemaEditor.NodeEditor.OpenGraph (Schema.Graph graphObj) (at ./Library/PackageCache/com.acdamiani.schema@15b26ec482/Editor/Window/NodeEditor.cs:133)
SchemaEditor.NodeEditor.Open (System.Int32 instanceID, System.Int32 line) (at ./Library/PackageCache/com.acdamiani.schema@15b26ec482/Editor/Window/NodeEditor.cs:278)
UnityEngine.GUIUtility:ProcessEvent(Int32, IntPtr, Boolean&)
```

Blackboard entries are still there but conditionals and connections seem to be gone at least visually.

[BaseAIGraph.asset.zip](https://github.com/acdamiani/schema/files/14035477/BaseAIGraph.asset.zip)
[AI.zip](https://github.com/acdamiani/schema/files/14035479/AI.zip)

I've added my Graph Asset and my custom Action and Conditional for you to reproduce it.

![image](https://github.com/acdamiani/schema/assets/20422129/267697b0-c3b2-404e-a9fd-1bcc9b88f5b5)

After going back to v 1.0.5 from Asset Store I still get the same error and I cannot create new connections. :-(

**Comment by ijisthee** [comment]
@acdamiani to avoid misunderstanding I did not update vom 1.0.5 to https://github.com/acdamiani/schema/commit/4d65500b8deb3f4127e8d2d4b06802d7da5b0545 or https://github.com/acdamiani/schema/commit/627c8558169341cae22a89872df20d41795ed027 but only from 1.0.5 to https://github.com/acdamiani/schema/commit/15b26ec4829f64418e47cd99410415604fe16ac7

**Comment by ijisthee** [comment]
I'm having a theory. It could be that I've created a class inside of the Schema folder structure and used it in the Graph asset. To update from Asset Store Package to Git Package I've had to delete the Schema folder from Assets folder. Unity automatically places it inside the Root/Packages folder.

Then obviously I've lost my class and the Graph Asset could not be opened anymore.
My Class was a conditional and the error happened in 

`SchemaEditor.Internal.ComponentSystem.Components.ConditionalComponent.Create...`

Since the class is gone and I cannot put it back to the Root/Packages/Schema folder since it's readonly I cannot get the Graph asset to work.

Maybe it is possible in the future to simply create a warning (missing GraphObject) or so.

Without that the Graph asset is completely gone and a lot of work is lost eventually.
